cilium-cli/connectivity/builder/builder.go

```diff
@@ -300,6 +300,7 @@ func renderTemplates(param check.Parameters) (map[string]string, error) {
 		"clientEgressL7HTTPMatchheaderSecretPortRangeYAML": clientEgressL7HTTPMatchheaderSecretPortRangeYAML,
 		"clientEgressL7HTTPExternalYAML":                   clientEgressL7HTTPExternalYAML,
 		"clientEgressNodeLocalDNSYAML":                     clientEgressNodeLocalDNSYAML,
+		"clientEgressOnlyDNSPolicyYAML":                    clientEgressOnlyDNSPolicyYAML,
```

Comment:
This is nice! Unrelated to the PR (more a proposal for a follow-up), we should also feature gate the NodeLocalDNS and OpenShift exceptions (if we can detect them properly)

Reply:
Yep, makes sense, give that they are not needed in most cases (I'm not sure about how feature detection works for them, but I guess we already have something for NodeLocalDNS given that it has dedicated tests). I'm going to mark this comment as resolved for the moment, to allow merging this PR.